.github/workflows/build.yml

```diff
@@ -0,0 +1,31 @@
+name: "Build"
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ${{matrix.os}}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v2.4.0
+        with:
+          # Nix Flakes doesn't work on shallow clones
+          fetch-depth: 0
+      - uses: cachix/install-nix-action@v15
+        with:
+          install_url: https://releases.nixos.org/nix/nix-2.5.1/install
+          extra_nix_config: |
+            experimental-features = nix-command flakes
+      - name: nix build
+        run: |
+          nix build
+      - name: nix log
+        if: ${{ always() }}
```

Comment:
Line can be removed?

Reply:
see below, we need this when builds fail

Reply:
I meant the if condition, not the entire step.